src/pages/atomicdex/api/v20/index.mdx

```diff
@@ -214,17 +205,220 @@ The `EvmNode` object includes the following items for a given coin or token:
+</Note>
+
+### LightningChannelOptions
+
+
+| Parameter                              | Type    | Description                                                                                   |
+|----------------------------------------|---------|-----------------------------------------------------------------------------------------------|
+| proportional_fee_in_millionths_sats    | integer | Optional, defaults to `0`. Amount (in milli-satoshi) charged for payments forwarded outbound over the channel, in excess of proportional_fee_in_millionths_sats. |
+| base_fee_msat                          | integer | Optional, defaults to `1000`. Amount (in milli-satoshi) charged for payments forwarded outbound over the channel, in excess of proportional_fee_in_millionths_sats. |
+| cltv_expiry_delta                      | integer | Optional, defaults to `72`. Blocks until [CheckLockTimeVerify (CLTV)](https://academy.bit2me.com/en/que-es-cltv-bitcoin/) expiry.                                |
+| max_dust_htlc_exposure_msat            | integer | Optional, defaults to `5000000`. Limit our total exposure to in-flight [HTLCs](https://academy.binance.com/en/glossary/hashed-timelock-contract) which are burned to fees as they are too small to claim on-chain.  |
+| force_close_avoidance_max_fee_sats     | integer | Optional, defaults to `1000`. The additional fee we're willing to pay to avoid waiting for the counterparty's locktime to reclaim funds.                            |
+
+
+### LightningClosedChannelsFilter
+
+| Parameter                   | Type    | Description                                                                                                     |
+|-----------------------------|---------|-----------------------------------------------------------------------------------------------------------------|
+| channel_id                  | string  | Optional. Unique string identifying a channel by its ID.                                                        |
+| counterparty_node_id        | string  | Optional. A hexidecimal string identifying a counterparty node. |
+| funding_tx                  | string  | Optional. A transaction ID which added funds.                   |
+| from_funding_value          | integer | Optional. The minimum value of channel funding in satoshis. PARAM NAME NOT CONSISTENT.     |
+| to_funding_value            | integer | Optional. The maximum value of channel funding in satoshis. PARAM NAME NOT CONSISTENT.     |
+| channel_type                | string  | Optional. `Inbound` or `Outbound`. INCONSISTENT WITH `is_outbound` PARAM  |
+| closing_tx                  | integer | Optional. A transaction ID which closed the channel.    |
+| closure_reason              | integer | Optional. The reason a channel was closed.   |
+| claiming_tx                 | integer | Optional. A transaction ID which claimed funds from the channel.    |
```

Comment:
This description is right, it can be better written as "The ID of the transaction that returned the remaining outbound funds when the channel was closed to our on-chain address"

Reply:
thanks for the extra context; added.